flow.go

```diff
@@ -87,10 +87,10 @@ func (id ChainID) String() string {
 // entityHasher is a thread-safe hasher used to hash Flow entities.
 type entityHasher struct {
 	mut    sync.Mutex
-	hasher crypto.Hasher
+	hasher hash.Hasher
```

Comment:
Could these all stay as `crypto.Hash` and `crypto.Hasher`? 

I'd prefer to have the rest of the SDK files depend only on `flow-go-sdk/crypto`